frost/src/spatial.rs

```diff
@@ -0,0 +1,476 @@
+        "id": "air_temperature",
+        "name": "Air temperature",
+        "unit": "degC"
+        },
+        "station": {
+        "location": [
+        {
+        "from": "1927-08-01T00:00:00Z",
+        "to": "1975-12-01T00:00:00Z",
+        "value": {
+        "elevation(masl/hs)": "514",
+        "latitude": "59.984700",
+        "longitude": "10.669300"
+        }
+        },
+        {
+        "from": "1975-12-01T00:00:00Z",
+        "to": "1976-12-31T00:00:00Z",
+        "value": {
+        "elevation(masl/hs)": "514",
+        "latitude": "59.984700",
+        "longitude": "10.669300"
+        }
+        },
+        {
+        "from": "1997-09-16T00:00:00Z",
+        "to": "2012-01-04T00:00:00Z",
+        "value": {
+        "elevation(masl/hs)": "514",
+        "latitude": "59.984700",
+        "longitude": "10.669300"
+        }
+        },
+        {
+        "from": "2012-01-04T00:00:00Z",
+        "to": "9999-01-01T00:00:00Z",
+        "value": {
+        "elevation(masl/hs)": "514",
+        "latitude": "59.984700",
+        "longitude": "10.669300"
+        }
+        }
+        ],
+        "shortname": "Tryvannshøgda"
+        },
+        "timeseries": {
+        "geometry": {
+        "level": {
+        "unit": "m",
+        "value": "2"
+        }
+        },
+        "quality": {
+        "exposure": [
+        {
+        "from": "1943-01-01T06:00:00Z",
+        "to": "1943-01-01T06:00:00Z",
+        "value": "1"
+        },
+        {
+        "from": "1975-12-31T18:00:00Z",
+        "to": "1975-12-31T18:00:00Z",
+        "value": "unknown"
+        },
+        {
+        "from": "1997-09-16T00:00:00Z",
+        "to": "1997-09-16T00:00:00Z",
+        "value": "1"
+        }
+        ],
+        "performance": [
+        {
+        "from": "1943-01-01T06:00:00Z",
+        "to": "1943-01-01T06:00:00Z",
+        "value": "unknown"
+        }
+        ]
+        },
+        "timeoffset": "PT0H",
+        "timeresolution": "PT1H"
+        }
+        },
+        "available": {
+        "from": "1943-01-01T06:00:00Z"
+        }
+        },
+        "observations": [
+        {
+        "time": "2023-08-13T18:00:00Z",
+        "body": {
+        "qualitycode": "0",
+        "value": "13.3000002"
+        }
+        }
+        ]
+        }
+        ]
+        }
+}"#;
+
+    #[test]
+    fn test_json_to_spatial_cache() {
+        let resp = serde_json::from_str(RESP).unwrap();
+
+        let polygon: Vec<GeoPoint> = vec![
+            GeoPoint {
+                lat: 59.88,
+                lon: 10.57,
+            },
+            GeoPoint {
+                lat: 60.00,
+                lon: 10.58,
+            },
+            GeoPoint {
+                lat: 59.99,
+                lon: 10.83,
+            },
+            GeoPoint {
+                lat: 59.88,
+                lon: 10.87,
+            },
+        ];
+        let element: &str = r#"frost:air_temperature"#;
```

Comment:
Seems like I don't need this, since this was needed when I was actually testing making a call against frost and now we are just using the saved json resp.

Reply:
Left it commented out, in case we need to resurrect actually calling frost?

Reply:
No, delete it. You can get it from the git history anyway